packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -42,7 +45,15 @@ const badgeTypeClassNames = {
   [BADGE_TYPES.outline]: getClassName('bpk-badge--outline'),
 };
 
-const BpkBadge = props => {
+type Props = {
+  type: ?string,
```

Comment:
You already have `BADGE_TYPES` - so here you can do `$Keys<typeof BADGE_TYPES>`. Then, consumers will get a Flow error when they try to use an unrecognised one 🙌